worker/utils/utils.js

```diff
@@ -49,6 +58,23 @@ module.exports = {
     })
   },
 
+  async encryptJob(string1, string2){
+    const message = string1 + string2;
+    const hmac = crypto.createHmac('sha256', message);
```

Comment:
We need to use a password to create the Hmac. Then update the Hmac with the message. Example: https://nodejs.org/api/crypto.html#crypto_crypto_createhmac_algorithm_key_options.

The best way to manage the password is to use an environment variable which we will later set in kanopy using helm. For now you can assume that the tests will run with access to an environment variable named 'crypto_secret' or whatever you'd like to call it. An example of using environment variables in the worker pool: https://github.com/mongodb/docs-worker-pool/blob/master/worker/jobTypes/githubPushJob.js#L21